packages/plugin-vue/src/main.ts

```diff
@@ -143,10 +143,10 @@ export async function transformMain(
   // SSR module registration by wrapping user setup
   if (ssr) {
     output.push(
-      `import { useSSRContext } from 'vue'`,
+      `import { useSSRContext as _useSSRContext} from 'vue'`,
       `const _sfc_setup = _sfc_main.setup`,
       `_sfc_main.setup = (props, ctx) => {`,
-      `  const ssrContext = useSSRContext()`,
+      `  const ssrContext = _useSSRContext()`,
```

Comment:
We should name it `__vite_useSSRContext`, it is the convention for Vite internal helpers used in other places.

Note: I think we could detect the presence of useSSRContext and only inserted when needed, but this PR fixes the issue at this point and we could improve things later.